File: auth-service/src/main/java/ru/nand/authservice/entity/dto/RegisterDTO.java
package ru.nand.authservice.entity.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class RegisterDTO {
    @NotBlank(message = "Username can not be empty")
    private String username;

    @NotBlank(message = "Email must be in the correct format")
    private String email;

    @NotBlank(message = "Password can not be empty")
    private String password;

    private String requestId;

    private boolean emailVerified;

}

File: account-service/internal/models/dto/RegisterDto.go
package dto

// RegisterDto - Приходит от Auth-Service по HTTP
type RegisterDto struct {
	Username      string `json:"username"`
	Email         string `json:"email"`
	Password      string `json:"password"`
	RequestId     string `json:"requestId"`
	EmailVerified bool   `json:"emailVerified"`
}
